packages/e2e-tests/config/setup-debug-artifacts.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const util = require( 'util' );
+const root = process.env.GITHUB_WORKSPACE || __dirname + '/../../../';
```

Comment:
`__dirname + '/../../../';` - it won't work when consumed from `npm`. Would it be possible to use [`process.cwd()`](https://nodejs.org/api/process.html#process_process_cwd) instead?

Reply:
@gziolo I think it will work though as `__dirname` returns the path of `packages/e2e-tests/config`. But agreed - `process.cwd()` seems to be cleaner. Let's do that instead.

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/26957

Reply:
When you install this package from npm it lands inside `node_modules/@wordpress/e2e-tests` or it might be in a nested folder.

Reply:
ah that's a good point!